index.js

```diff
@@ -24,67 +25,99 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
-
+  let guess = parseInt(guessInput.value, 10);
+  console.log(guess)
+  
   hideAllMessages();
-
+  resetButton.style.display=''
+  
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
 
-    correctMessage.style.display = '';
+    
+    numOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+  
+    correctMessage.style.display ='';
+
 
     submitButton.disabled = true;
     guessInput.disabled = true;
+    resetButton.style.display=''
+    
   }
 
-  if (guess !== targetNumber) {
-    if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
-    } else {
-      tooLowMessage.style.display = '';
-    }
+  if (guess < targetNumber) {
+    tooLowMessage.style.display = '';
+  }
+  if (guess > targetNumber) {
+    tooHighMessage.style.display = '';
+  }
 
-    const remainingAttempts = maxNumberOfAttempts - attempts;
+  
 
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
-  }
 
-  if (attempts ==== maxNumberOfAttempts) {
-    submitButton.disabled = true;
-    guessInput.disabled = true;
+
+
+
+    const remainingAttempts = maxNumberOfAttempts - attempts;
+    console.log(remainingAttempts);
+    
+    if(guess === targetNumber){
+      submitButton.disabled = true;
+      guessInput.disabled = true;
+      guessInput.value = 'Submit Guess';
+      maxNumberOfAttempts.innerHTML="You guessed Correctly"
+      
   }
 
-  guessInput.value = '';
 
+
+  maxGuessesMessage.innerHTML = `You guessed ${guess} <br>   guesses remaining ${remainingAttempts}`;
+  maxGuessesMessage.style.display = '';
```

Comment:
Do we want to show this message in all cases? Or just when the guess was incorrect (ie: too high, or too low)? Right now because it's outside of the if statements, it will be shown on all guesses, even correct ones.

Reply:
it should show a message when the guess is too low or high and incorrect